### DISCUSSION TITLE
How do i even install llvm for this (pacman llvm does not work)?

### DISCUSSION THREAD

**Discussion by taitep** [TheDan64/inkwell#536] [Q&A] [answered]
I am trying to get this crate to work, but im getting an error when trying to just build a hello world with the crate installed. I think it says its missing some library files. I installed from pacman and the `llvm` and `llvm-libs` packages just dont include those files. Not sure if this is a problem in the repos, inkwell or somewhere completely else. Here is the full output from cargo: https://pastebin.com/FWnyt5AT

**Comment by kehrazy** [comment, marked as answer]
by building it manually with cmake.

**Comment by TheDan64** [comment]
Converting to a discussion as this is a question and not an issue.

LLVM has a setup installer bash script, I'm not sure if it works on arch(?) but it's worth looking into
